erizoAPI/WebRtcConnection.cc

```diff
@@ -150,8 +150,7 @@ NAN_METHOD(WebRtcConnection::New) {
     int maxPort = Nan::To<int>(info[6]).FromJust();
     bool trickle = Nan::To<bool>((info[7])).FromJust();
     Nan::Utf8String json_param(Nan::To<v8::String>(info[8]).ToLocalChecked());
-    bool use_nicer = Nan::To<bool>((info[9])).FromJust();
-    bool enable_connection_quality_check = Nan::To<bool>((info[10])).FromJust();
+    bool enable_connection_quality_check = Nan::To<bool>((info[9])).FromJust();
```

Comment:
There could be other parameters inside info that are related to the ICE configuration (turnServer, username, etc.)